velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -913,5 +913,69 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
       fromUnixTime(0, "yyyy-MM-dd HH:II"), "Specifier I is not supported");
 }
 
+TEST_F(DateTimeFunctionsTest, makeYMInterval) {
+  const auto getYMInterval = [](const std::optional<int32_t>& ymInterval) {
+    return ymInterval.has_value()
+        ? std::make_optional(INTERVAL_YEAR_MONTH()->valueToString(*ymInterval))
+        : std::nullopt;
+  };
+  const auto makeYMInterval = [&](const std::optional<int32_t>& year,
```

Comment:
perhaps,

makeYMInterval -> fromYearAndMonth
makeYMIntervalOnlyYear -> fromYear

for readability